docs/backends/postgresql.md

```diff
@@ -166,5 +166,7 @@ format of UUID on output. See the test `test_uuid_column_type_support` for usage
 To support older PostgreSQL versions, the following configuration macros are recognized:
 
 * `SOCI_POSTGRESQL_NOBINDBYNAME` - switches off the query rewriting.
-* `SOCI_POSTGRESQL_NOPARAMS` - disables support for parameterized queries (binding of use elements),automatically imposes also the `SOCI_POSTGRESQL_NOBINDBYNAME` macro. It is necessary for PostgreSQL 7.3.
+* `SOCI_POSTGRESQL_NOPARAMS` - disables support for parameterized queries (binding of use elements),automatically imposes also the
+* `SOCI_POSTGRESQL_NOBINDBYNAME` macro. It is necessary for PostgreSQL 7.3.
```

Comment:
The description of macro `SOCI_POSTGRESQL_NOBINDBYNAME` is duplicated

Reply:
Good catch. I created new bullet by accident.